docs/changelog.md

```diff
@@ -24,7 +24,13 @@ permalink: /changelog/
 
 * **feature-push**
   * Simplified error handling and reduced non-fatal exception reporting.
+ 
+* **lib-crash**
+  * ⚠️ **This is a breaking change**: breadcrumbs now have `support-base` dependency.
```

Comment:
The wording on this changelog entry is not super clear, I think. Just from reading this I wouldn't be sure what that means. 🤔 
I also wonder if we should make `support-base` an `api` dependency in `build.gradle` so that a consumer of `lib-crash` does not have to import `support-base` manually to get access to breadcrumbs. `support-base` is something that helps us internally but an end-user should not have to bother with it.

Reply:
Ok, I'll remove this entry.